Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM golang:1.22-alpine AS builder
+
+WORKDIR /app
+
+COPY go.mod .
+COPY go.sum .
+COPY ./src ./src
+
+RUN go mod download
+RUN go mod verify
+
+RUN go build -ldflags="-w -s" -o server ./src/server
+
+FROM scratch
+COPY --from=builder app/server bin/
+
+ENTRYPOINT [ "/bin/server" ]
```

Comment:
it's minor but I recommend you configure your editor/IDE to always have (exactly) one newline at the end of files (vi does it by default but so can VS Code etc) that would avoid that ⛔ you see in the diff view

Reply:
Thanks for noticing this. I will revert.